packages/aseprite/aseprite.pacscript

```diff
@@ -27,7 +27,7 @@ prepare() {
   unzip -qq "Skia-Linux-Release-x64-libc++.zip" -d skia
 
   # Apply patch from AUR
-  patch -stp1 <PACSTALL_patchesdir/"desktop.patch?h=aseprite"
+  patch -stp1 <PACSTALL_patchesdir/desktop.patch
```

Comment:
Turns out this doesn't work as the pacstall PR had intended. 

Now, curl -sO is split between two versions:
on jammy and below, it still produces `desktop.patch?h=aseprite`
on bookworm and above, it produces `desktop.patch`

Your pacscript is now going to have to figure out how to conform to this.